docs/resources/cluster_profile.md

```diff
@@ -142,6 +144,20 @@ resource "spectrocloud_cluster_profile" "profile" {
 }
 ```
 
+### Example of Providing Multiple Packs
+
+You can provide multiple packs at once by leveraging a dynamic block.  
+
+!> The order of the Packs must be taken into consideration so avoid any situations where the order of the Packs is re-arranged by Terraform, such as nested loops `{ for pack in var.packs : pack => pack }`. We recommend creating a variable that contains the list of Packs arranged in order of the App Profile stack and their respective configuration.
```

Comment:
Can you make the example `for_each = { for pack in var.packs : pack => pack }`

Reply:
@kreeuwijk I'm not sure I am following. Are you wanting the code snippet to have the following?
```terraform
resource "spectrocloud_cluster_profile" "this" {
  name = "security-profile"
  dynamic "pack" {
    for_each = { for pack in var.packs : pack => pack }
  }
```

Reply:
No I mean changing

"...such as nested loops `{ for pack in var.packs : pack => pack }`. We recommend..."
to
"...such as nested loops `for_each = { for pack in var.packs : pack => pack }`. We recommend..."

Reply:
Fixed ✅